app/population/index.js

```diff
@@ -0,0 +1,33 @@
+const router = require('express').Router({ mergeParams: true })
```

Comment:
I'm sure you need the `mergeParams` option in this instance as doesn't look like any sub-routers are being created?

Reply:
I did need it in the previous iteration, but I've stripped out all the sub-routers for this PR - so I'll remove it.